### DISCUSSION TITLE
欢迎各位加入ToolDiy

### DISCUSSION THREAD

**Discussion by storage-db** [cargo-youth/ToolDiy#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SakurajimaMaii** [comment]
来了来了

**Comment by qzpaper** [comment]
来了来了

**Comment by Origin715** [comment]
了解GitHub从ToolDiy开始_(:з」∠)_

**Comment by storage-db** [comment]
是的你是对的嘿嘿
